app/Http/Controllers/PhotosController.php

```diff
@@ -400,18 +400,16 @@ public function unverified ()
     {
         $user = Auth::user();
 
-        $photos = Photo::select('id', 'filename', 'lat', 'lon', 'model', 'remaining', 'display_name', 'datetime')
-            ->where([
-                'user_id' => $user->id,
-                'verified' => 0,
-                'verification' => 0
-            ])->simplePaginate(1);
-
-        $remaining = Photo::where([
+        $query = [
             'user_id' => $user->id,
             'verified' => 0,
             'verification' => 0
-        ])->count();
+        ];
+
+        $photos = Photo::select('id', 'filename', 'lat', 'lon', 'model', 'remaining', 'display_name', 'datetime')
+            ->where($query)->simplePaginate(1);
+
+        $remaining = Photo::where($query)->count();
```

Comment:
Can you change this to

```
$query = Photo::where([
            'user_id' => $user->id,
            'verified' => 0,
            'verification' => 0
]);

$photos = $query->select('id', 'filename', 'lat', 'lon', 'model', 'remaining', 'display_name', 'datetime')->simplePaginate(1);

$remaining = $query->count();
```

Reply:
I think you should not do that because it will get all data and then select and pagination. That will create a performance issue.

Reply:
I understand your concern but thankfully it won't get a large data object. 1. `$query` will simply create the query without getting any data. 2. `->simplePaginate(1);` will get 1 object, that has an array of 1 photos. 3. we can use the same `$query` to count the table rows, without actually loading all the data

Reply:
Updated. Thanks